src/pyasm/search/sql_dumper.py

```diff
@@ -345,6 +356,24 @@ def dump_tactic_inserts(my, path, mode='sql'):
 
             data = sobject.get_data()
             for name, value in data.items():
+                
+                if my.replace_dict:
+                    
+                    for column,replace_args in my.replace_dict.iteritems():
+                        if name == column:
+                            import re
+                            project_code = replace_args[0]
+                            replace_str = replace_args[1]
+                            regex = replace_args[2]
+                            
+                            if not re.match(regex,value):
```

Comment:
raise TacticException instead of Exception.. ALso mention what the suggested format is.. Give the user some guidance.